code/game/objects/structures/door_assembly.dm

```diff
@@ -7,8 +7,9 @@
 	max_integrity = 200
 	var/overlays_file = 'icons/obj/doors/airlocks/station/overlays.dmi'
 	var/state = AIRLOCK_ASSEMBLY_NEEDS_WIRES
-	var/mineral
-	var/mineral_type
+	var/mineral // String value. Used in user chat messages
+	var/mineral_type // mineral object path.
```

Comment:
```suggestion
	/// String value. Used in user chat messages
	var/mineral
	/// mineral object path.
	var/mineral_type
```
This will ensure autodoc will pick the comments up